llvm/lib/Transforms/IPO/FunctionAttrs.cpp

```diff
@@ -654,7 +654,15 @@ determinePointerAccessAttrs(Argument *A,
       // must be a data operand (e.g. argument or operand bundle)
       const unsigned UseIndex = CB.getDataOperandNo(U);
 
-      if (!CB.doesNotCapture(UseIndex)) {
+      // Some intrinsics (for instance ptrmask) do not capture their results,
+      // but return results thas alias their pointer argument, and thus should
```

Comment:
```suggestion
      // but return results that alias their pointer argument, and thus should
```

Reply:
... Apparently I forgot to `git commit --amend` in the typo fix. Should I make a new PR for that or?